src/main/java/com/skytask/service/ProductService.java

```diff
@@ -0,0 +1,25 @@
+package com.skytask.service;
+
+import com.skytask.model.Product;
+import com.skytask.repository.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
+
+    private ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public List<Product> getList() {
+        return productRepository.findAll();
+    }
+
+    public void create(Product product) {
+        productRepository.insert_product(product.getName(), product.getDescription(), product.getPrice(), product.getStock());
```

Comment:
Is it possible to pass the product object only and make the magic in the repository interface?